registry/storage/util.go

```diff
@@ -0,0 +1,21 @@
+package storage
+
+import (
+	"github.com/docker/distribution/context"
+	"github.com/docker/distribution/registry/storage/driver"
+)
+
+// Exists provides a utility method to test whether or not a path exists in
+// the given driver.
+func exists(ctx context.Context, drv driver.StorageDriver, path string) (bool, error) {
```

Comment:
```
$ go build
# github.com/docker/distribution/registry/storage
./tagstore.go:58: undefined: exists
```

This is used by `tagstore`. It was in `blobstore` but was moved out. There may be some future refactoring in `tagstore` so this will go away.